client/me/purchases/manage-purchase/index.jsx

```diff
@@ -157,10 +157,59 @@ const ManagePurchase = React.createClass( {
 		);
 	},
 
+	getExpiringText( purchase ) {
+		if ( purchase.expiryStatus === 'manualRenew' ) {
+			return (
+				this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s. ',
```

Comment:
Alternate string suggestion: 
* `translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s.' )`&mdash; translations: **20**. *ES Score: 9.95*

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plans-improved-renew-warnings)